packages/govuk-frontend/src/govuk/components/skip-link/skip-link.mjs

```diff
@@ -7,7 +7,7 @@ import { GOVUKFrontendComponent } from '../../govuk-frontend-component.mjs'
  */
 export class SkipLink extends GOVUKFrontendComponent {
   /**
-   * @protected
+   * @override
    * @type {HTMLAnchorElement}
    */
   $module = this.$module
```

Comment:
Odd, but that's how class fields work. Without that initialisation, `this.$module` would be set to `undefined` after the `super()` call in the `constructor()`... which would kind of defeat the purpose of encapsulating its setting in the base class.

More info [on what happens around these class fields on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Public_class_fields#examples).